### PR TITLE
io.dbeaver.DBeaverCommunity.Client.mariadb extension for DBeaver

### PR DESCRIPTION
**Pull request by krashekspres**
extension for dbeaver

### CODE REVIEW

File: io.dbeaver.DBeaverCommunity.Client.mariadb.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
+{
+  "app-id": "io.dbeaver.DBeaverCommunity.Client.mariadb",
+  "runtime": "io.dbeaver.DBeaverCommunity",
+  "sdk": "org.freedesktop.Sdk//18.08",
```
Comment (nedrichards) [inline io.dbeaver.DBeaverCommunity.Client.mariadb.json]: could you show this as a separate runtime-version please?

**Comment by nedrichards** [comment]
this needs a separate flathub.json file that limits it only x86-64 like the parent project. https://github.com/flathub/io.dbeaver.DBeaverCommunity/blob/master/flathub.json

**Comment by nedrichards** [comment]
Can you also include appdata please, then software stores will be able to correctly link and show these extensions to be installed inline.

**Comment by bilelmoussaoui** [comment]
Documentation for appdata files for addons https://www.freedesktop.org/software/appstream/docs/sect-Metadata-Addon.html

File: io.dbeaver.DBeaverCommunity.Client.mariadb.json (hunk 1 above)
Comment (krashekspres) [reply io.dbeaver.DBeaverCommunity.Client.mariadb.json]: Sorry, I can't figure out what do you want :)
add `runtime-version` tag for SDK?

Comment (p1u3o) [reply io.dbeaver.DBeaverCommunity.Client.mariadb.json]: These really were not ready for pull yet. That's why I didn't make the pulls myself :)

https://github.com/flathub/io.dbeaver.DBeaverCommunity/pull/23#issuecomment-500238332

I'll have a look at them again in the morning.

Comment (p1u3o) [reply io.dbeaver.DBeaverCommunity.Client.mariadb.json]: @krashekspres Please see krashekspres/flathub#1

**Comment by bilelmoussaoui** [comment]
io.dbeaver.DBeaverCommunity.Client.mariadb

**Comment by krashekspres** [comment]
bot, build io.dbeaver.DBeaverCommunity.Client.mariadb

**Comment by krashekspres** [comment]
bot, build io.dbeaver.DBeaverCommunity.Client.mariadb

**Comment by krashekspres** [comment]
bot, build io.dbeaver.DBeaverCommunity.Client.mariadb

**Comment by krashekspres** [comment]
bot, build io.dbeaver.DBeaverCommunity.Client.mariadb

**Comment by nedrichards** [comment]
Repository has been created: https://github.com/flathub/io.dbeaver.DBeaverCommunity.Client.mariadb

**Pull request closed without merging**